crates/rslint_parser/src/syntax/module.rs

```diff
@@ -746,6 +766,9 @@ fn parse_export_default_clause(p: &mut Parser) -> ParsedSyntax {
 
     let clause = match p.nth(1) {
         T![class] => parse_export_default_class_case(p),
+        T![ident] if p.nth_src(1) == "abstract" && p.nth_at(2, T![class]) => {
```

Comment:
Nit: I started to prefer using `is_at_contextual_keyword` so that we can correctly handle the unicode sequences when we have incremental parsing 
```suggestion
        T![ident] if is_nth_at_contextual_keyword(p, 1, "abstract") && p.nth_at(2, T![class]) => {
```